src/Symfony/Component/Debug/Exception/FlattenException.php

```diff
@@ -33,16 +33,30 @@ class FlattenException
     private $file;
     private $line;
 
-    public static function create(\Exception $exception, $statusCode = null, array $headers = array())
+    /**
+     * @param \Throwable $throwable
+     * @param int|null   $statusCode
+     * @param array      $headers
+     *
+     * @return static
+     */
+    public static function create($throwable, $statusCode = null, array $headers = array())
```

Comment:
your https://github.com/symfony/symfony/pull/26447#discussion_r174090182 comment applies as well, removing a typehint breaks childs

Reply:
You're right, it will raise a warning. I'll add another deprecation then.